### PR TITLE
Create the indie journey

### PR DESCRIPTION
**Pull request by DeyvidJLira**

### CODE REVIEW

**Comment by josegallo** [comment]
May be you should ask for help to @casey-collab ;)

File: portugues/jornada-indie/O inicio.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Seja um desenvolvedor independente de jogos agora mesmo:
```
Comment (casey-collab) [inline portugues/jornada-indie/O inicio.md]: It looks like this page is not reachable from the rest of the story. That is, if a reader starts in `language.md` and keeps clicking links, they will never find this page. You can fix this by finding a good place in the existing story to add an option with a link that leads to this page.

**Comment by casey-collab** [comment]
Besides my other comments, this looks good!  Once you respond to those, this should be good to merge.

File: portugues/jornada-indie/O inicio.md (hunk 1 above)
Comment (DeyvidJLira) [inline portugues/jornada-indie/O inicio.md]: Oh sorry, okay! I add the link now! Thanks!

**Comment by josegallo** [comment]
@casey-collab

**Comment by casey-collab** [comment]
Looks great! :ribbon:

**Pull request merged**